docs/ops/arithmetic/Tanh_1.md

```diff
@@ -4,17 +4,22 @@
 
 **Category**: *Activation function*
```

Comment:
Since the op category is Activation function I suggest to move it to activation dir.
You have an example [here](https://github.com/openvinotoolkit/openvino/pull/5039/files)

Reply:
Will do.